### PR TITLE
Created default splits for Fluo table that are now set during init

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

File: modules/data/src/main/java/io/fluo/webindex/data/spark/IndexEnv.java [hunk 1]
```diff
@@ -110,14 +118,23 @@ public void initAccumuloIndexTable(SortedSet<Text> splits) {
...
     } catch (AccumuloException | AccumuloSecurityException | TableNotFoundException e) {
       throw new IllegalStateException("Failed to add splits to Accumulo table " + table, e);
     }
   }
 
-  public void initAccumuloIndexTable() {
-    initAccumuloIndexTable(getDefaultSplits());
+  public void setFluoTableSplits(SortedSet<Text> splits) {
+    final String table = fluoConfig.getAccumuloTable();
+    if (conn.tableOperations().exists(table)) {
```
Comment (keith-turner) [inline modules/data/src/main/java/io/fluo/webindex/data/spark/IndexEnv.java]: check is not really needed.  Split will throw TableNotFound if it does not exists.

Comment (mikewalch) [inline modules/data/src/main/java/io/fluo/webindex/data/spark/IndexEnv.java]: fixed in 0b39b4b2042278

**Comment by mikewalch** [comment]
Also added kryo to app classpath in 5ff169f2aafa9

**Comment by keith-turner** [comment]
+1

**Pull request merged**